assets/sass/modules/_consent.scss

```diff
@@ -17,6 +17,7 @@
     left: 0;
     right: 0;
     top: 0;
+    pointer-events: none;
```

Comment:
What does this do?

Reply:
if it's not there the border div seems to block the image and make it non-clickable. Added this with advice from @manueltanzi-okta

Reply:
not quite IE friendly.

Reply:
forgot to put the link http://caniuse.com/#feat=pointer-events

Reply:
@haishengwu-okta oh thank you, my bad. @danielzeller-okta so maybe we should move the images inside consent-beacon-border and fix the padding issue.